### PR TITLE
Simplify multiple support statements with just notes for webextensions/

### PR DESCRIPTION
**Pull request by queengooborg**
This PR is a cherry-pick of #17006 for the unreviewed parts of the `webextensions/` folder.

### CODE REVIEW

**Comment by queengooborg** [comment]
I'm going to merge this since it's highly unlikely there are any issues.  Happy to follow up with changes in the future.

**Review by foolip — COMMENTED** [review]
I had a pending comment.

File: webextensions/api/extension.json [hunk 1]
```diff
@@ -171,15 +171,10 @@
...
-                    "version_added": "93",
-                    "notes": "Sidebar views are now matched."
-                  },
-                  {
-                    "version_added": "45"
-                  }
-                ],
+                "firefox": {
+                  "version_added": "45",
+                  "notes": "Since Firefox 93, sidebar views are now matched."
```
[the remark this answers is not in the document]
Comment (queengooborg) [reply webextensions/api/extension.json:176]: I have no clue, honestly!

**Comment by Angelstjes** [comment]
> I had a pending comment.

**Comment by Angelstjes** [comment]
> I'm going to merge this since it's highly unlikely there are any issues.  Happy to follow up with changes in the future.

**Review by Angelstjes — APPROVED** [review]
>

File: webextensions/api/extension.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (foolip) [reply webextensions/api/extension.json:176]: Okay, I also don't know, and it's hard to judge if this note is useful then 🤔

**Pull request merged**